libs/components/indicators/src/lib/modules/illustration/illustration-resolver.service.ts

```diff
@@ -6,7 +6,13 @@ import { Injectable } from '@angular/core';
 @Injectable()
 export abstract class SkyIllustrationResolverService {
   /**
-   * Resolves a URL for the specified illustration name.
+   * Resolves a URL for the specified illustration name to render in an `img`.
    */
   public abstract resolveUrl(name: string): Promise<string>;
+
+  /**
+   * Resolves the `href` of the SVG element to reference in `use`.
+   * If both an `href` and a URL are resolved, the SVG with `href` will be rendered.
+   */
+  public abstract resolveHref(name: string): Promise<string>;
```

Comment:
Adding an abstract method to an abstract class is a breaking change because [abstract methods must be overridden by inherting classes](https://www.typescriptlang.org/docs/handbook/2/classes.html#abstract-classes-and-members). To make it optional, you need to provide a default implementation.

```suggestion
  public async resolveHref(name: string): Promise<string> {
    return '';
  }
```